test_requirements.txt

```diff
@@ -3,6 +3,7 @@ pytest-cov~=2.10.1
 pytest~=6.1.0
 pytest-rerunfailures~=9.1.1
 pytest-xdist~=2.1.0
+pytest-mock~=3.3.1
```

Comment:
is it not part of pytest already since some version?

Reply:
I don't think so